src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DateValue.txt

```diff
@@ -261,4 +261,7 @@ Date(2045,2,3)
 Date(45,2,3)
 
 >> DateValue("02 03, 0045 04:56:07")
-Date(45,2,3)
\ No newline at end of file
+Date(45,2,3)
+
+>> DateValue("10/1/1920","vi-vn")
+Date(1920,1,10)
```

Comment:
We should have tests for different languages. DateValue should be the reverse of a call to Text(<date>, DateTimeFormat.ShortDate, <language>).

You can try running something like the expression below to find more test cases:

    Concat(
        [
            "bg-BG", "ca-ES", "cs-CZ", "da-DK", "de-DE",
            "el-GR", "en-US", "es-ES", "et-EE", "eu-ES",
            "fi-FI", "fr-FR", "gl-ES", "hi-IN", "hr-HR",
            "hu-HU", "id-ID", "it-IT", "ja-JP", "kk-KZ",
            "ko-KR", "lt-LT", "lv-LV", "ms-MY", "nb-NO",
            "nl-NL", "pl-PL", "pt-BR", "pt-PT", "ro-RO",
            "ru-RU", "sk-SK", "sl-SI", "sr-Cyrl-RS",
            "sr-Latn-RS", "sv-SE", "th-TH", "tr-TR",
            "uk-UA", "vi-VN", "zh-CN", "zh-TW"
        ],
        Value & ": " & Text(Date(2021, 4, 9), DateTimeFormat.ShortDate, Value),
        Char(10))